users/src/index.ts

```diff
@@ -0,0 +1,75 @@
+import Express, { Request, Response, NextFunction } from "express"
+import { MikroORM, type PostgreSqlDriver } from "@mikro-orm/postgresql"
+
+import { applicationDefault, initializeApp } from "firebase-admin/app"
+import { DecodedIdToken, getAuth } from "firebase-admin/auth"
+import { env } from "process"
+import CORS from "cors"
+
+import ProfileRouter from "./profile"
+import ActivityRouter from "./activity"
+
+declare global {
+    namespace Express {
+        interface Request {
+            firebaseToken: DecodedIdToken
+            orm: MikroORM
+        }
+    }
```

Comment:
Ooh, didn't know this worked. Am i correct to say that this means all Express.Request objects now have these properties? Cool, TIL.